lib/IAMClient.js

```diff
@@ -545,8 +545,9 @@ class IAMClient {
                 return callback(null, obj, res.statusCode, res.statusMessage);
             }
             // Load the error from errors(arsenal)
-            return callback(
-                errors[obj.ErrorResponse.Error.Code], null, res.statusCode);
+            const retErr = obj.ErrorResponse
```

Comment:
Is it to handle the case where getObj returns through it's callback an empty object (e.g. {}) ?